### ISSUE TITLE
Application took about 2.5 sec to start

### ISSUE THREAD

**Issue by coolit** [gtk-rs/gtk#853]
Just wondering is this startup time normal or not?

**Comment by sdroege** [comment]
Not normal, no. What application on which OS on which hardware with which GTK version? :)

**Comment by coolit** [comment]
All applications in the examples, win10, cpu intel 2.4G, GTK version is as required by the examples I guess.

**Comment by EPashkin** [comment]
Seems this normal on windows with antivirus, IMHO because debug binaries is big (>150MB),
there also some delay when application tried to connect DBus 2 times and decide to "emulate/host" it yourself.

**Comment by coolit** [comment]
The release binaries are about 12.3MB, took the same time to start as the debug ones. After re-testing, the delay was found to be about 1.5 sec, whch is acceptable though.

**Comment by neivv** [comment]
A few years ago I had a ~10 second startup time on Windows 10 that was caused by GTK calling a win32 cursor loading function for all/most files in `share\icons\Adwaita\cursors`. Apparently something in Windows 10 caused that function run a lot slower than you'd expect.

I worked around the issue by renaming the `cursors` directory so that GTK won't find any custom cursors.

Maybe this is the same issue? Though today it seems that having `cursors` is adding "only" a second to my startup time, so it seems that Microsoft already mostly fixed what performance issues I was seeing.
